.github/workflows/release.yaml

```
@@ -0,0 +1,52 @@
+name: Release
+on:
+  push:
+    tags:
+    - 'v*'
+env:
+  go-version: 1.16
+  tag: ${GITHUB_REF#refs/tags/v}
+jobs:
+  image:
+    name: Push Container Image
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - run: docker build -t quay.io/cybozu/pod-security-admission:latest .
+      - name: Push docker image to Quay.io
+        run: |
+          echo ${{ secrets.QUAY_PASSWORD }} | docker login -u ${{ secrets.QUAY_USER }} --password-stdin quay.io
+          echo "pushing image ..."
+          docker tag quay.io/cybozu/pod-security-admission:latest quay.io/cybozu/pod-security-admission:${{ env.tag }}
+          docker push quay.io/cybozu/pod-security-admission:${{ env.tag }}
+          BRANCH=$(echo ${{ env.tag }} | cut -d "." -f 1-2)
+          docker tag quay.io/cybozu/pod-security-admission:latest quay.io/cybozu/pod-security-admission:$BRANCH
+          docker push quay.io/cybozu/pod-security-admission:$BRANCH
+  release:
+    name: Release on GitHub
+    needs: image
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - run: make build/install.yaml
```

```suggestion
      - run: make setup
      - run: make build/install.yaml
```

Possibly missing `make setup`.

```
/path/to/pod-security-admission/bin/kustomize build ./config/default > build/install.yaml
/bin/bash: /path/to/pod-security-admission/bin/kustomize: No such file or directory
make: *** [Makefile:77: build/install.yaml] Error 127
```

---

`kustomize:` should be `$(KUSTOMIZE):`.